### ISSUE TITLE
withTranslation for pages

### ISSUE THREAD

**Issue by Svyatoslav16** [vinissimus/next-translate#300]
I tried to find the useTranslation analogue for the class component and found it in the npm examples, but this trap works only for the components, not for the whole page. This is probably not what I was looking for, but I found nothing but an npm example and links to it in https://github.com/vinissimus/next-translate/issues.

Please tell us how to do the right thing, thank you in advance for your help

```jsx
import React from 'react';
import Link from 'next/link';
import withTranslation from 'next-translate/useTranslation';

class SetupDemo extends React.Component {
    constructor(props) {
        super(props);
        };
    }

    render() {
        const { i18n } = this.props;
        return (
            <div>simple text</div>
        );
    }
}

export default withTranslation(SetupDemo);
```

**Comment by aralroca** [comment]
@Svyatoslav16 I have detected a bug passing from 0.19 when loading the namespaces when there was the HOC `withTranslation`. I have fixed it and it will be ready for version **0.19.3*. In theory, this should work:



```js
import React from 'react'
import withTranslation from 'next-translate/withTranslation'

class Page extends React.Component {
  constructor(props) {
    super(props);
  };

  render() {
    const { t } = this.props.i18n;
    const title = t`common:title`

    return <div>{title}</div>
  }
}

export default withTranslation(Page)
```

I made a prerelease of [**0.19.3-canary.1**](https://github.com/vinissimus/next-translate/releases/tag/0.19.3-canary.1) with this fix. Can you confirm if it works now? Thanks a lot 🙏 !

**Comment by Svyatoslav16** [comment]
It's works, thanks
